tests/angular/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think it would be good to have a bit of documentation explaining the pattern of having tests in separate packages since I think it's more common to expect tests next to the code. Some ideas:
1. fill in the `description` field for each test package to say something like "This is an unpublished package which contains the tests for ..."
2. Add a README for each test package saying the same thing
3. Add a README at the root of the tests folder describing the pattern/rationale
4. Add a note in the CONTRIBUTING doc explaining the testing strategy

I don't think we need all of these but a few sign posts would help.

Reply:
Added docs to Contributing.md, READMEs pointing to the docs, and description fields to the package.json files